source/connect/tls.txt

```diff
@@ -0,0 +1,245 @@
+
+.. _ruby-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS for the connection to your MongoDB deployment in the following ways:
+
+- Set the ``ssl`` option to ``true`` in a new ``Mongo:Client`` object
+- Set the ``tls`` option to ``true`` in your connection string
+
+.. note:: SSL Naming Convention
+
+    All {+mdb-server+} versions supported by the Ruby driver (2.6 and higher) 
+    only implement TLS. 2.6 and do not use SSL.
+    
+    For historical reasons, the Ruby option names pertaining to TLS configuration 
+    use the ``ssl`` rather than the ``tls`` prefix. The next major version of the Ruby 
+    driver (3.0) will use the ``tls`` prefix for Ruby option names.
+
+.. tabs::
+
+    .. tab:: Mongo::Client
+        :tabid: mongoclient
+
+        To configure certificates, you must specify the following options:
+
+        - ``ssl_cert``: The certificate file used to identify the connection against MongoDB.
+        - ``ssl_key``: The private keyfile used to identify the connection against MongoDB.
+        - ``ssl_ca_cert``: The file containing the concatenated CA certificates 
+          used to validate certs passed from the other end of the connection.
+          By default, the driver will use the default system root certificate 
+          store as the trust anchor, if the option is not designated. 
+
+        These certificate options can be specified as different types. For example, the 
+        ``ssl_cert`` option has the alternative ``ssl_cert_object`` and ``ssl_cert_string`` 
+        options, to specify the certificate file as either a certificate object or a string.
+        For more information on certificate options, see the Options Hash section of 
+        the API Documentation for `Mongo::Client <{+api-root+}/Mongo/Client.html>`__ .  
+
+        In the following example, the TLS certificate and corresponding private key 
+        are provided in separate files:
+
+        .. literalinclude:: /includes/connect/tls.rb
+            :language: ruby
+            :dedent:
+            :start-after: start-enable-tls-settings
+            :end-before: end-enable-tls-settings
+        
+        The TLS certificate and private key can be contained in the same file, but 
+        both the certificate and private key options must still be specified. In the 
+        following example, the TLS certificate and the private key are both defined 
+        in the same ``client.pem`` file:
+
+        .. literalinclude:: /includes/connect/tls.rb
+            :language: ruby
+            :dedent:
+            :start-after: start-enable-tls-settings-same-file
+            :end-before: end-enable-tls-settings-same-file
+
+    .. tab:: Connection String
+        :tabid: uri
+
+        To configure certificates, you must specify the following URI options:
+        
+        - ``tlsCertificateKeyFile``: The certificate and key file used to identify
+          identify the connection against MongoDB. Both the certificate and key
```

Comment:
q: is 'identify the connection against mongodb' the right language? i would have gone for 'identify the client to MongoDB' or something similar.

Reply:
'identify the connection against mongodb' is the language used in the API documentation, but it is a bit strange sounding. Changed to 'verify the connection to MongoDB' .